### PR TITLE
feat: add Intel Ethernet Controller I225-V driver

### PR DESCRIPTION
**Pull request by smira**
https://linux-hardware.org/?id=pci:8086-15f3-1043-87d2

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

**Review by Ulexus — APPROVED** [review]

File: kernel/build/config-arm64 [hunk 1]
```diff
@@ -2784,7 +2784,7 @@ CONFIG_IAVF=y
 CONFIG_I40EVF=y
 # CONFIG_ICE is not set
 # CONFIG_FM10K is not set
-# CONFIG_IGC is not set
+CONFIG_IGC=y
```
Comment (frezbo) [inline kernel/build/config-arm64:2787]: do we need these NUC drivers for ARM?

**Review by frezbo — APPROVED** [review]

File: kernel/build/config-arm64 (hunk 1 above)
Comment (smira) [reply kernel/build/config-arm64:2787]: we tried to keep configs in sync across arches, if not something which can't be on a specific platform

I don't know if there's any ARM computer with that network card, but that's possible I guess?

Comment (frezbo) [reply kernel/build/config-arm64:2787]: yeh, makes sense to keep both in sync

**Pull request merged**